.gitignore

```diff
@@ -10,4 +10,6 @@ a.out
 roms/
 third_party/fmt
 bios.bin
-*.sav
\ No newline at end of file
+*.sav
+*.swp
```

Comment:
This is a vim thing right? We don't need to commit it in .gitignore

Reply:
.sav files are GBA save files, but .swp are vim

Reply:
i removed .swp